packages/concerto-cto/test/cto/aliasedImport.cto

```diff
@@ -0,0 +1,9 @@
+namespace test.aliasing
+
+import org.accordproject.{doc1 as d1,doc2 as d2}
```

Comment:
These are test cases. I have only added new test case file .  

So basically we first parse the cto file first to get the AST(json) then this AST is converted back using the Printer. 
Then we check if the original cto string is equal to the new obtained cto string. 

Since we checking the cto strings at the end ,which has also been done in the existing testcases. The output format of the Printer is essential. Thus, I have kept the output format of the Printer and input format of the cto file(test case) the same. 

That is why you don't see the spaces.

Reply:
Are there any unit tests where the cases can be added?